src/main/java/com/example/onlinebookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.example.onlinebookstore.repository;
+
+import com.example.onlinebookstore.model.Book;
+import java.util.List;
+
+public interface BookRepository {
+    Book save(Book book);
+
+    List findAll();
```

Comment:
Do not use raw classes
```suggestion
    List<BookDto> findAll();
```

Reply:
As far as I know I should implement Dto in today's task